package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "4.3.2-dev",
+  "version": "4.2.4-dev",
```

Comment:
```suggestion
  "version": "4.3.2-dev",
```

Reply:
Yup, fixed. Ran through the actual steps just now to check that they work. Our bump2version wasn't getting package-lock.json which confused it.